articles/kotlin-basics/index.md

```diff
@@ -0,0 +1,1701 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/kotlin-basics/
+title: Kotlin Programming for Beginners 
+description: Learn Kotlin Programming, its basics, and Fundamentals from scratch.
+author: stephen-chacha
+date: 2021-01-12T00:00:00-14:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/kotlin-basics/hero.png
+    alt: kotlin basics image
+---
+
+#### Introduction
+Learn Kotlin Programming, its basics, and Fundamentals from scratch. 
+
+#### Prerequisites
+For you to follow along with this tutorial, you need to have:
+- Kotlin editor preferably [IntelliJ](https://www.jetbrains.com/idea/download/#section=windows) installed on your machine.
+
+### Topics to be covered
+1. Data Types and Variables 
+    - Data Types and Variables
```

Comment:
Just mention the topic of discussion to make your TOC as short as possible. The rest shall be covered in that section.